### PR TITLE
#115756483   Create an API link that leads to /developer

### PR DESCRIPTION
**Pull request by ademola-raimi**
#### What does this PR do?
- Create an index page for developers page.
#### Description of Task to be completed?
- Create an API link that leads to /developers  route
- Design the page of the /developer which is the home page of the suyabay API 
#### How should this be manually tested?
- The user scrolls down to the footer to click the API link beside Copyright. This will direct the user to the API index page (/developer)
- In the Index page, There are 4 menus (developer, docs, partner and myApp), the user clicks any of these menus, it will direct the user to a different URL (/developer for developer, /developer/docs for docs, /developer/partners for partner and /developer/myApp for myApp).
- The different URL will display a 404 error except the developer menu. This is because the story for other menus has not been written yet.
- Check the below screenshot for a clearer picture:

screen shot 2016-03-23 at 5 04 51 pm

### CODE REVIEW

**Comment by unicodeveloper** [comment]
Please take care of Nitpick CI issues and why are your questions formatted that way? Did you copy it yourself?..Didn't it grab the questions for the Pull request template that @andela-cvundi did?

File: app/Http/Controllers/ApiController.php [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Suyabay\Http\Controllers;
+
+class ApiController extends Controller
```
Comment (unicodeveloper) [inline app/Http/Controllers/ApiController.php]: Why is it named `ApiController` ?

**Comment by ademola-raimi** [comment]
I named it APiController because It will control all the developer's pages

**Comment by unicodeveloper** [comment]
Meet @andela-jkariuki or @andela-cvundi for guidance on how to properly write the answers to the questions raised in the PR template. `Description of the task to be completed`'s answer is not correct, same goes for `How should this be manually tested?`, the answer for that is very very vague @andela-tolotin

**Comment by unicodeveloper** [comment]
Is this ready to be merged now @andela-araimi ?

**Comment by unicodeveloper** [comment]
Please rebase @andela-araimi

**Comment by unicodeveloper** [comment]
Please rebase @andela-araimi

**Comment by unicodeveloper** [comment]
Take care of all the design inconsistencies on the design of the page, The `docs` and `myapp` have their first letter in small letter and the font is not appropriate for the title of the tabs. Please change it to youtube style font @andela-araimi . Attention to details is needed for a great design

**Comment by unicodeveloper** [comment]
Please remove the search functionality on that page. it's not needed.

**Comment by ademola-raimi** [comment]
You can merge these now @Prosper.

**Comment by unicodeveloper** [comment]
Why do your commits have no github avatar? @andela-araimi

**Comment by ademola-raimi** [comment]
@prosper: I will check my GitHub setting to fix it

**Pull request merged**